src/cpu/aarch64/xbyak_aarch64/xbyak_aarch64/xbyak_aarch64_util.h

```diff
@@ -97,10 +84,6 @@ class Cpu {
   void dumpCacheInfo() const;
   Arm64CacheType getCacheType(const Arm64CacheLevel i) const;
   uint32_t getCoresSharingDataCache(const Arm64CacheLevel i) const;
-  /* 2023.02.11 */
-#if defined(__GNUC__) || defined(__clang_version__)
-  uint32_t __attribute__((deprecated("Please use getLastDataCacheLevel()"))) getDataCacheLevels() const;
-#endif
   uint32_t getDataCacheSize(const Arm64CacheLevel i) const;
   const char *getImplementer() const;
   uint32_t getLastDataCacheLevel() const;
```

Comment:
[random point] Could you copy [xbyak_aarch64_perf.h](https://github.com/fujitsu/xbyak_aarch64/blob/7d436dadce322c1b9b7a0bddb5ea1a51421e7215/xbyak_aarch64/xbyak_aarch64_perf.h) into `src/cpu/aarch64/xbyak_aarch64/xbyak_aarch64`? At this time, it does not affect oneDNN, but it is a piece of Xbyak_aarch64.

Reply:
done